src/web3/evm/connector/erc20/actions/allowance.ts

```diff
@@ -12,7 +12,7 @@ export async function allowanceAction(input: ConnectorInput<unknown>): Promise<A
     await sanitizeParameters({
       ...input,
       fields: {
-        ...fields,
+        ...(fields as any),
```

Comment:
@SAPikachu In this case, it is specific connectors (ERC20, Chainlink, Syndicate...). So I think we can know the specific types and create them to avoid `any` casting, but this PR is already very big, we will open other PRs on a case-by-case basis, do you agree?

Reply:
Agreed 👍